app/assets/main/react_components/contexts/UserActionsContext.js

```diff
@@ -110,6 +111,23 @@ export const UserActionsProvider = ({
     setCategoryBadges(badge);
   };
 
+  const memberShipBadge = {
+    id: "-1",
+    name: "Climate Friend",
+    badge_image_url: "/achievement_images/AchievementClimateFriend.png",
+    userActionsArray:
+      JSON.parse(userSubscriptionType) == true
+        ? [
```

Comment:
Maybe we can avoid duplicate code by using something like:

status: JSON.parse(userSubscriptionType == true ? true : false

instead of two similar array objects?